pylint_django/transforms/transforms/django_utils_translation.py

```diff
@@ -1,2 +1,5 @@
-def ugettext_lazy(_):
+def gettext_lazy(_):
     return ''
+
+
+ugettext_lazy = gettext_lazy
```

Comment:
can you disable the invalid-name pylint check here. pylint expects this to be all caps because it doesn't understand this is supposed to be a function.